Composer/packages/client/src/pages/botProject/PublishTargets.tsx

```diff
@@ -116,6 +138,15 @@ export const PublishTargets: React.FC<PublishTargetsProps> = (props) => {
                     {formatMessage('Edit')}
                   </ActionButton>
                 </div>
+                <div css={tableRowItem(columnSizes[2])}>
+                  <ActionButton
+                    data-testid={'deletePublishProfile'}
+                    styles={editPublishProfile}
+                    onClick={() => onDeletePublishTarget(p)}
```

Comment:
what's the point of use callback if it's used in an anonymous function (re-defines with every render)

Reply:
Good point.